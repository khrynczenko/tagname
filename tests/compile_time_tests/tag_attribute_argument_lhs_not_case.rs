use tagname::TagName;

#[derive(TagName)]
enum Cases {
    #[tag(something = "something")]
    Lower,
    Upper
}

fn main() {
}
