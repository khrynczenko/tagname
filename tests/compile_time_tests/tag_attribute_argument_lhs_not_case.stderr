error: proc-macro derive panicked
 --> tests/compile_time_tests/tag_attribute_argument_lhs_not_case.rs:3:10
  |
3 | #[derive(TagName)]
  |          ^^^^^^^
  |
  = help: message: case value must be either "upper" or "lower"
